core/tests/dispatch_integration_tests.rs

```diff
@@ -1,12 +1,83 @@
 use kompact::{prelude::*, prelude_test::net_test_helpers::*};
-use std::{net::SocketAddr, thread, time::Duration};
+use std::{net::SocketAddr, sync::Arc, thread, time::Duration};
 
 fn system_from_network_config(network_config: NetworkConfig) -> KompactSystem {
     let mut cfg = KompactConfig::default();
     cfg.system_components(DeadletterBox::new, network_config.build());
     cfg.build().expect("KompactSystem")
 }
 
+fn create_and_run_lazy_pinger(
+    system: &KompactSystem,
+    target_ponger: &ActorPath,
+) -> (Arc<Component<PingerAct>>, KFuture<()>) {
+    let mut pinger_act = PingerAct::new_lazy(target_ponger.clone());
+    let pong_future = pinger_act.make_future();
+    let (pinger, reg_future) = system.create_and_register(move || pinger_act);
+    reg_future.wait_expect(Duration::from_millis(1000), "Pinger failed to register!");
+    system.start(&pinger);
+    (pinger, pong_future)
+}
+
+fn create_and_run_eager_pinger(
+    system: &KompactSystem,
+    target_ponger: &ActorPath,
+) -> (Arc<Component<PingerAct>>, KFuture<()>) {
+    let mut pinger_act = PingerAct::new_eager(target_ponger.clone());
+    let pong_future = pinger_act.make_future();
+    let (pinger, reg_future) = system.create_and_register(move || pinger_act);
+    reg_future.wait_expect(Duration::from_millis(1000), "Pinger failed to register!");
```

Comment:
I think should start avoiding these local duration literals in tests wherever possible and instead use constants at the module or at least fields at the test level. If we at some point figure out that a duration was poorly chosen it'll be a shitton of work to change all the usages. Now we still have a good opportunity to nip this problem in the butt.

Reply:
Yeah that makes a lot of sense, thanks.